include/seqan3/alphabet/structure/structured_rna.hpp

```diff
@@ -0,0 +1,203 @@
+    using char_type = underlying_char_t<sequence_alphabet_type>;
+    //!\brief Equals the char_type of the structure_alphabet_type.
+    using str_char_type = underlying_char_t<structure_alphabet_type>;
+
+    /*!\name Write functions
+     * \{
+     */
+    //!\brief Directly assign the sequence character.
+    constexpr structured_rna & operator=(sequence_alphabet_type const l) noexcept
+    {
+        get<0>(*this) = l;
+        return *this;
+    }
+
+    //!\brief Directly assign the structure character.
+    constexpr structured_rna & operator=(structure_alphabet_type const l) noexcept
+    {
+        get<1>(*this) = l;
+        return *this;
+    }
+
+    //!\brief Assign from a nucleotide character. This modifies the internal sequence letter.
+    constexpr structured_rna & assign_char(char_type const c)
+    {
+        seqan3::assign_char(get<0>(*this), c);
+        return *this;
+    }
+
+    //!\brief Assign from a structure character. This modifies the internal structure letter.
+    constexpr structured_rna & assign_structure(str_char_type const c)
+    {
+        seqan3::assign_char(get<1>(*this), c);
+        return *this;
+    }
+    //!\}
+
+    /*!\name Read functions
+     * \{
+     */
+    //!\brief Return the structure character. This reads the internal structure letter.
+    constexpr str_char_type to_structure() const noexcept
+    {
+        return seqan3::to_char(get<1>(*this));
+    }
+
+    //!\brief Return a character. This reads the internal sequence letter.
+    constexpr char_type to_char() const noexcept
+    {
+        return seqan3::to_char(get<0>(*this));
+    }
+    //!\}
+
+    /*!\name RNA structure properties
+     * \{
+     */
+    constexpr bool is_pair_open() const noexcept
+    {
+        return get<1>(*this).is_pair_open();
+    };
+
+    constexpr bool is_pair_close() const noexcept
+    {
+        return get<1>(*this).is_pair_close();
+    };
+
+    constexpr bool is_unpaired() const noexcept
+    {
+        return get<1>(*this).is_unpaired();
+    };
+
+    //!\brief The ability of the alphabet to represent pseudoknots, i.e. crossing interactions.
+    static constexpr bool pseudoknot_support{structure_alphabet_t::pseudoknot_support};
+    //!\}
+};
+
+//!\brief Type deduction guide enables usage of structured_rna without specifying template args.
+//!\relates structured_rna
+template <typename sequence_alphabet_type, typename structure_alphabet_type>
+structured_rna(sequence_alphabet_type &&, structure_alphabet_type &&)
+    -> structured_rna<std::decay_t<sequence_alphabet_type>, std::decay_t<structure_alphabet_type>>;
+
+} // namespace seqan3
+
+namespace seqan3::detail
+{
+
+//!\brief Since seqan3::structured_rna wraps a nucleotide alphabet it is also one.
+template <typename sequence_alphabet_type, typename structure_alphabet_type>
+struct is_nucleotide<structured_rna<sequence_alphabet_type, structure_alphabet_type>> : public std::true_type
+{};
```

Comment:
this has been removed. instead the existence of a complement function is the requirement for the nucleotide concept. *please rebase your PR on the current master branch*
I am not really sure if the compositions should actually satisfy the individual concepts. If the individual concepts are required people should maybe just convert 🤔

Reply:
anyway, if you do want to satisfy nucleotide_concept, you need to add a complement member function to the composition.